.circleci/config.yml

```diff
@@ -84,12 +95,9 @@ commands:
         type: string
         default: "" # if unset, e2e-test.sh specifies the command
     steps:
-      - checkout
-      - run: ./scripts/assert-changed-files.sh "<< parameters.trigger_pattern >>|<< parameters.test_path >>/*"
-      - <<: *restore_cache
-      - <<: *install_node_modules
-      - <<: *persist_cache
       - <<: *attach_to_bootstrap
```

Comment:
This is great! :clap: Just wondering if this will work if we ever clear the cache as e2e-test aren't part of the monorepo and they need a separate npm install.

Reply:
hmm. i really don't know. Any suggestions on how to test?

Reply:
use your own circleci github repo I guess but i'm fine with just commenting

```
      # In case of failure, add these steps again. Cache probably got deleted
      #- <<: *restore_cache
      #- <<: *install_node_modules
      #- <<: *persist_cache
```